test/test_serialization_utils.py

```diff
@@ -58,3 +59,10 @@ def test_serialize_timestamp(self):
         datetime_object = datetime.datetime.strptime(datetime_str, "%Y-%m-%d")
         result = _serialize_json_safe(datetime_object)
         assert datetime_str in result
+
+    def test_serialize_via_json_timestamp(self):
+        timestamp_obj = pd.Timestamp(2020, 1, 1)
+        assert isinstance(timestamp_obj, pd._libs.tslibs.timestamps.Timestamp)
```

Comment:
Is there a public version of `pd._libs.tslibs.timestamps.Timestamp` we can use?

Reply:
maybe pd.tslib.Timestamp?
https://stackoverflow.com/questions/25852044/converting-pandas-tslib-timestamp-to-datetime-python

Reply:
Looks like it is https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Timestamp.html

Reply:
Updated.